crates/rethnet_eth/Cargo.toml

```diff
@@ -16,6 +16,7 @@ primitive-types = { version = "0.11.1", default-features = false, features = ["r
 reqwest = { version = "0.11", features = ["blocking", "json"] }
 revm-primitives = { git = "https://github.com/Wodann/revm", rev = "9fdf446", version = "1.1", default-features = false }
 # revm-primitives = { path = "../../../revm/crates/primitives", version = "1.0", default-features = false }
+ruint = { version = "1.7.0", default-features = false }
```

Comment:
`revm-primitives` re-exports `ruint`,  so there is no need to add it as a dependency. Using re-exports avoids us growing out of sync with `revm` and introducing compiler errors.

https://docs.rs/revm-primitives/latest/revm_primitives/index.html

Reply:
addressed in https://github.com/NomicFoundation/hardhat/pull/4218/commits/6d1e0ba16866a04340148603d71fd79c6a95023b